yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java

```diff
@@ -0,0 +1,71 @@
+package com.yoti.api.client.docs.session.create;
+
+/**
+ * Represents the configuration properties for the web/native
+ * SDK clients for a given Doc Scan session.
+ * <p>
+ * Doc Scan sessions can be configured with different web/native
+ * SDK settings allowing configuration of some visual and functional
+ * properties.
+ *
+ * @since 2.7
+ */
+public interface SdkConfig {
+
+    /**
+     * Retrieves the allowed capture methods from the SDK configuration
+     *
+     * @return the allowed capture methods
+     */
+    String getAllowedCaptureMethods();
+
+    /**
+     * Retrieves the primary colour from the SDK configuration
+     *
+     * @return the primary colour
+     */
+    String getPrimaryColour();
```

Comment:
Was going to say we should probably be consistent with the spelling of "colour" , as we have `ApplicationProfile.application_receipt_bgcolor`, but since this is reflecting the JSON (`primary_colour`), I'm thinking it would probably be better to leave it as it is